### ISSUE TITLE
[A11Y]: Accessibility audit round 2 issues

### ISSUE THREAD

**Issue by davidmenendez** [carbon-design-system/carbon-ai-chat#1220]
### High severity

**SL No: 22 - Page settings effect in sidebar need to be more clear**

From the document:

> Selecting options from Page Setup dropdowns triggers unexpected changes in layout, visual appearance, and content. Such sudden modifications may make the page inaccessible for assistive technology users.

> "When users interact with Page Setup dropdowns:
> 1. Selecting orientation options (e.g., Left-to-Right) changes component positioning, impacting navigation sequence.
> 2. Theme selection alters visual presentation, affecting users with low vision and color perception issues.
> 3. Selecting certain options (e.g., “True” in Code Changes) dynamically inserts additional content in the chat area (beginning and end).
> These changes are triggered immediately upon selection without prior indication or user confirmation, resulting in unexpected context changes."

Because options like `Text Direction` cause major shifts in the layout of the page, there should be additional mechanisms in place to inform the user about the effects of their selection.

> "Make the Page set up mechanisum more understandable /predictable  by opting  following:
> 1. Clearly indicate in advance, through accessible instructions (programmatically associated), that selecting dropdown options will trigger layout, visual, or content changes.
> 2. Prevent accidental activation by introducing a second level of confirmation (e.g., Are you sure confirmation dialog, with Yes /No Buttons before applying  major changes).

> "Screen reader and other assistive aid users  users may experience unexpected navigation changes and lose their current position due to layout shifts. Users with low vision or cognitive disabilities may find sudden visual or content changes disorienting, making the interface difficult to understand and operate."

It sounds like we need to either include additional static language that informs the user about these changes or we need to add a confirmation mechanism that informs the user of the changes before proceeding.

### Low severity

**SL No: 23 - Text direction options not descriptive enough**

`LTR` and `RTL` should not be abbreviated in order to assist users who are unfamiliar with the acronym
